origin-notifications/app.js

```diff
@@ -50,8 +50,13 @@ app.use(bodyParser.json())
 // currently showing all subscriptions in an unauthenticated index view - maybe not a good idea in production?
 app.get('/', async (req, res) => {
   const subs = await PushSubscription.findAll()
```

Comment:
I'd recommend moving the DB query within the (app.get('env') === 'development') statement.